### ISSUE TITLE
Implement new parameter and people attribute "intent_to_use"

### ISSUE THREAD

**Issue by pausz** [fpsim/fpsim#242]
- Create dummy data to add a new empowerment attribute capturing the "intent to use a contraceptive method"
#### Intialisation
- Could be stratified by age 
- Could be stratified by setting (rural)  

### Update rule
- Could change over time due to other demographic and empowerment related attributes

**Comment by MObrien-IDM** [comment]
@pausz Is this resolved with your new PR?

**Comment by pausz** [comment]
@MObrien-IDM This has not been implemented yet. PR #346 implements fertility intent (#337).

**Comment by mzimmermann-IDM** [comment]
This is one of the issues that we will have to complete before finishing the empowerment module. This is only applicable for women NOT currently using contraction. This can be initiated based on this csv of intent to use contraception by age. https://github.com/fpsim/fpsim/blob/Kenya_empowerment_DHS/fpsim/locations/kenya/contra_intent.csv. @pausz does this make sense?

For the update rule, this will be update once a year on her birthday based on a function of fertility intent, age, parity, edu, wealth, and urban/rural. Coefficients are here: https://github.com/fpsim/fpsim/blob/Kenya_empowerment_DHS/fpsim/locations/kenya/intent_coef.csv
@robynstuart can you help with this part?
